.github/labeler.yml

```
@@ -0,0 +1,76 @@
+
+# API/Backend
+api:
+  - changed-files:
+    - any-glob-to-any-file:
+      - 'netlify/functions/**/*'
+      - 'src/lib/api/**/*'
+
+# Accessibility
+accessibility:
+  - changed-files:
+    - any-glob-to-any-file:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)